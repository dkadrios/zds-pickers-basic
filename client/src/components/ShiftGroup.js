import React from 'react'
import GridControls from './entries/GridControls'
import GridHeader from './entries/GridHeader'
import ShiftEntriesGrid from './entries/ShiftEntriesGrid'

const ShiftGroup = props => (
  <section>
    <GridControls {...props} />
    <GridHeader {...props} />
    <ShiftEntriesGrid {...props} />
  </section>
)

export default ShiftGroup
