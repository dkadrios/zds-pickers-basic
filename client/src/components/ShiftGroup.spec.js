import ShiftGroup from './ShiftGroup'

const props = {
  groupId: 0,
  sortOn: 0,
  sortBy: 'SORT_BY_ALL',
  sortDir: 0,
  changeSort: f => f,
  editEntry: f => f,
  removeEntry: f => f,
  selectShiftEntry: f => f,
  entries: [],
  disabled: false,
  selectAllEntries: f => f,
  clearAllSelections: f => f,
  handleEntryClick: f => f,
}

describe('FactoryReset tests', () => {
  it('renders correctly', () => {
    shallowExpect(ShiftGroup)(props).toMatchSnapshot()
  })
})
