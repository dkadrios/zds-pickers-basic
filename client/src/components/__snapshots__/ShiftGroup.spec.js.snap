// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FactoryReset tests renders correctly 1`] = `
<section>
  <WithStyles(GridControls)
    changeSort={[Function]}
    clearAllSelections={[Function]}
    disabled={false}
    editEntry={[Function]}
    entries={Array []}
    groupId={0}
    handleEntryClick={[Function]}
    removeEntry={[Function]}
    selectAllEntries={[Function]}
    selectShiftEntry={[Function]}
    sortBy="SORT_BY_ALL"
    sortDir={0}
    sortOn={0}
  />
  <GridHeader
    changeSort={[Function]}
    clearAllSelections={[Function]}
    disabled={false}
    editEntry={[Function]}
    entries={Array []}
    groupId={0}
    handleEntryClick={[Function]}
    removeEntry={[Function]}
    selectAllEntries={[Function]}
    selectShiftEntry={[Function]}
    sortBy="SORT_BY_ALL"
    sortDir={0}
    sortOn={0}
  />
  <ShiftEntriesGrid
    changeSort={[Function]}
    clearAllSelections={[Function]}
    disabled={false}
    editEntry={[Function]}
    entries={Array []}
    groupId={0}
    handleEntryClick={[Function]}
    removeEntry={[Function]}
    selectAllEntries={[Function]}
    selectShiftEntry={[Function]}
    sortBy="SORT_BY_ALL"
    sortDir={0}
    sortOn={0}
  />
</section>
`;
